soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -342,6 +342,19 @@ void Randomizer::LoadHintLocations(const char* spoilerFileName) {
         "Warp to&{{location}}?\x1B&%gOK&No%w\x02",
         "Warp to&{{location}}?\x1B&%gOK&No%w\x02", // TODO: German translation
         "Se téléporter vers&{{location}}?\x1B&%gOK!&Non%w\x02" });
+
+    CustomMessageManager::Instance->CreateMessage(Randomizer::hintMessageTableID, TEXT_LAKE_HYLIA_WATER_SWITCH_SIGN,
```

Comment:
I threw the custom sign/navi text in the `hintMessageTableID` but I suppose these aren't "hints". Not sure a better spot for it. It's not clear to me how we want the different "message tables", or why we would need different tables to begin with.